.github/workflows/tests.yml

```diff
@@ -0,0 +1,40 @@
+name: Tests
+
+on:
+  push:
+    branches: '*'
+  pull_request:
+    branches: '*'
+
+jobs:
+  tests:
+
+    runs-on: ubuntu-latest
```

Comment:
Should we also do a matrix on OS too?

Reply:
I usually wouldn't bother for a pure-Python package unless there's some particular reason to think its behaviour will differ between platforms. But it should be easy to do - at least for Mac - if you'd like.